mandatory/2-function-creation.js

```diff
@@ -20,15 +20,25 @@ Complete the function to check if the variable `num` satisfies the following req
 Tip: use logical operators
 */
 
-function validate(num) {}
+function validate(num) {
+  let answer;
+  if (num <= 100 && num % 2 === 0) {
```

Comment:
You should include a test to see if its a number

Reply:
Hi @sirpfaira my first condition ``if (num<=100 && num%2==0) `` I believe can only be performed on a number and would give false if a string was given as the first condition would not be met.